demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/03/Version20240313120508.php

```diff
@@ -22,7 +22,8 @@ class Version20240313120508 extends AbstractMigration
 {
     public function getDescription(): string
     {
-        return 'refs T36340: Extract phase of an procedure into own entity. Step2: Copy current data into new entity.';
```

Comment:
I think these changes are not intended for this pr?

Reply:
yes ok, but doesn't matter its only an formatting improvement, which also can be done here.